vfs/dir.go

```diff
@@ -188,13 +188,24 @@ func (d *Dir) invalidateDir(absPath string) {
 //
 // if entryType is a directory it invalidates the parent of the directory too.
 func (d *Dir) changeNotify(relativePath string, entryType fs.EntryType) {
-	defer log.Trace(d.path, "relativePath=%q, type=%v", relativePath, entryType)("")
 	d.mu.RLock()
 	absPath := path.Join(d.path, relativePath)
 	d.mu.RUnlock()
-	d.invalidateDir(vfscommon.FindParent(absPath))
+
+	defer log.Trace(d.path, "relativePath=%q, absPath=%q, type=%v", relativePath, absPath,
```

Comment:
This is obviously a test, however if you want to change the VFS code then it is probably material for another PR!